NEWS.md

```diff
@@ -4,6 +4,12 @@ ClimaCore.jl Release Notes
 main
 -------
 
+- ![][badge-🤖precisionΔ] ![][badge-💥breaking] `Remapper`s can now process
```

Comment:
Machine precision changes are a bit surprising to me, but I'm still looking at the PR

Reply:
The structure of the GPU kernels changed a little bit and I am not sure that everything will be bitwise identical.